### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by alexandrmiee**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+              - GET
+              - HEAD
+              - OPTIONS
+              - PATCH
+              - POST
+              - PUT
+            ## The origin id defined above
+            TargetOriginId: WebApp
+            ## Defining if and how the QueryString and Cookies are forwarded to the origin which in this case is S3
+            ForwardedValues:
```
Comment (Savsan) [inline serverless.yml:103]: This field is deprecated.
Read this: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-defaultcachebehavior.html#cfn-cloudfront-distribution-defaultcachebehavior-forwardedvalues

**Comment by Savsan** [comment]
Where is the links to S3 and CloudFront?

**Comment by EzhiG** [comment]
> Where is the links to S3 and CloudFront?

`+`

**Comment by Savsan** [comment]
I see no them...

**Comment by alexandrmiee** [comment]
Sorry, forgot about it
S3 - http://lukovnikovan-nodejs-aws-fe-cloudfront.s3-website-eu-west-1.amazonaws.com/
Cloudfront - https://d3u0pdfp2lxn9o.cloudfront.net/

**Comment by Savsan** [comment]
Nice! Well done!

**Comment by EzhiG** [comment]
Still cannot score, CloudFront looks like this:
image

**Pull request merged**